src/headers.js

```diff
@@ -2,11 +2,12 @@
 // @id           wasabee.user.js
 // @name         IITC plugin: Wasabee Draw Tools
 // @namespace    https://wasabee.rocks/
-// @version      0.12.1
-// @updateURL    https://wasabee.rocks/wasabee.meta.js
-// @downloadURL  https://wasabee.rocks/wasabee.user.js
+// @version      0.13.2-deviousness
```

Comment:
You probably don't want to check in this change.

Reply:
indeed.